lti_provider/mixins.py

```diff
@@ -24,6 +24,12 @@ def join_groups(self, request, lti, ctx):
                 request.user.groups.add(ctx.faculty_group)
                 break
 
+            if settings.LTI_TOOL_CONFIGURATION['allow_ta_access'] is False:
```

Comment:
Your initial approach to this felt, e.g.

`if settings.LTI_TOOL_CONFIGURATION['allow_ta_access'] and 'teachingassistant' in role:`

What was your thinking around changing it?

Reply:
No reason! While debugging I rearranged a lot. I missed changing this back.